### PR TITLE
20_ros_widh_dds.md: minor wording, what is dds

### PR DESCRIPTION
**Pull request by scpeters**

### CODE REVIEW

File: articles/20_ros_with_dds.md [hunk 1]
```diff
@@ -39,9 +39,10 @@ DDS provides a publish-subscribe transport which is very similar to ROS's publis
 DDS uses the "Interface Description Language (IDL)" as defined by the [Object Management Group (OMG)](http://www.omg.org/) for message definition and serialization.
 DDS does not yet provide a request-response style transport, which would be like ROS's service system, but a draft for that style of transport is being reviewed right now (2014).
 
-Supporting the publish-subscribe transport, DDS provides a completely distributed discovery system by default.
-Because of this, any two DDS programs can communicate without the need for an equivalent tool like the ROS master.
-This makes the system more fault tolerant and flexible, but it is not required to use the dynamic discovery mechanism, and so different DDS vendors provide options for static discovery.
+By supporting the publish-subscribe transport, DDS provides a completely distributed discovery system by default.
```
Comment (wjwwood) [inline articles/20_ros_with_dds.md]: This should be more like "In order to support publish-subscribe, DDS has a discovery system which happens to be distributed".

I feel like neither my sentence nor the one you proposed really conveys the relationship correctly.

Comment (scpeters) [inline articles/20_ros_with_dds.md]: I updated the line: "In order to support publish-subscribe transport, DDS provides a distributed discovery system by default."

File: articles/20_ros_with_dds.md [hunk 2]
```diff
@@ -39,9 +39,10 @@ DDS provides a publish-subscribe transport which is very similar to ROS's publis
 DDS uses the "Interface Description Language (IDL)" as defined by the [Object Management Group (OMG)](http://www.omg.org/) for message definition and serialization.
 DDS does not yet provide a request-response style transport, which would be like ROS's service system, but a draft for that style of transport is being reviewed right now (2014).
 
-Supporting the publish-subscribe transport, DDS provides a completely distributed discovery system by default.
-Because of this, any two DDS programs can communicate without the need for an equivalent tool like the ROS master.
-This makes the system more fault tolerant and flexible, but it is not required to use the dynamic discovery mechanism, and so different DDS vendors provide options for static discovery.
+In order to support publish-subscribe transport, DDS provides a distributed discovery system by default.
```
Comment (wjwwood) [inline articles/20_ros_with_dds.md]: Yeah, that's still not quite right.

I think it should be more like "The default discovery system provided by DDS, which is required to use DDS's publish-subscribe transport, is a distributed discovery system."

Comment (scpeters) [inline articles/20_ros_with_dds.md]: Sounds good. I pasted your recommendation.

**Comment by wjwwood** [comment]
Awesome, thanks for the proofreading.

**Pull request merged**